firefox-ios/Client/Frontend/Home/Pocket/PocketViewModel.swift

```diff
@@ -7,7 +7,8 @@ import Foundation
 import Storage
 import Shared
 
-class PocketViewModel {
+// TODO: - FXIOS-12756
```

Comment:
```suggestion
// TODO: FXIOS-12756 Should be a main actor synchronized data model, not Sendable
```
I like to add a little context to the TODO 🙂

Reply:
right totally agree. Just one think @ih-codes wouldn't be better to have this model conform to `Sendable` instead of tighin it to @MainActor i think it can be more flexible solution as this class as only one mutable state that can be moved else where and just refer to telemetry